features/step_definitions/common_stepdef.rb

```diff
@@ -4,7 +4,7 @@
   visit '/sign-in'
   page.driver.browser.manage.add_cookie(name: 'crown_marketplace_cookie_settings_viewed', value: 'true')
   value = common.header_one.text
-  sign_in if value == 'Sign in to your account'
+  sign_in if value == 'Sign in to your legal services buyer account'
 end
```

Comment:
This step_def will now only work when logging in to legal services and will fail for MC and ST.
Perhaps we need to rethink what is the best way of logging in when there are multiple services?
We can discuss this later